test/tools/ossfuzz/protoToAbiV2.cpp

```diff
@@ -220,16 +231,108 @@ std::string ProtoConverter::fixedByteValueAsString(unsigned _width, unsigned _co
 	//      <-----------total length --------->
 	// Note: This assumes that maskUnsignedIntToHex() invokes toHex(..., HexPrefix::Add)
 	unsigned startPos = 66 - numMaskNibbles;
+	// Extracts the least significant numMaskNibbles from the result
+	// of maskUnsignedIntToHex().
+	return maskUnsignedIntToHex(
+		_counter,
+		numMaskNibbles
+	).substr(startPos, numMaskNibbles);
+}
+
+std::string ProtoConverter::hexValueAsString(
+	unsigned _numBytes,
+	unsigned _counter,
+	bool _isHexLiteral,
+	bool _decorate
+)
+{
+	solAssert(_numBytes <= 32, "Proto ABIv2 fuzzer: Invalid hex length");
```

Comment:
As far as I understand I can't be `0` because of the assertion in `variableLengthValueAsString` right? So maybe also add it here for clarity?